src/Bootstrap/ExtensionConfigResolver.php

```diff
@@ -11,11 +11,12 @@
 final class ExtensionConfigResolver
 {
     /**
-     * @param SmartFileInfo[] $configFileInfos
      * @return SmartFileInfo[]
```

Comment:
Btw, the `SmartFileInfo` can return only existing files. If the file is missing, the exception is thrown. This is to avoid false positive on incorrect file location.

That means the `not_found` you've commened on is already checked here :+1:

Reply:
I just wanted to say the same thing right now.